docs/js/main.js

```diff
@@ -56,6 +55,22 @@ const showRecipes = function (recipes) {
     for (const recipe in recipes) {
         const recipeID = recipes[recipe].id;
         const title = recipes[recipe].name;
+        //show array of tags for each recipe
+        const allRecipeTags = recipes[recipe].tags;
+        //console.log(allRecipeTags);
+        //show each tag object
+        for (let tag in allRecipeTags) {
+            //console.log(tag, allRecipeTags[tag]);
+            let allTags = (tag, allRecipeTags[tag]);
```

Comment:
I'm not sure what this line means...

Reply:
I definitely think I got into overkill land here. So I was taught that in order to loop through multiple objects and access their key value pairs, you would need to put all of the objects in an array. So I overkilled it by trying to console out each tag THEN force it into an array (no successfully), which required a loop. I ALREADY have an array of the tags in allRecipe tags x.x